device-registry/src/main/scala/org/genivi/sota/device_registry/Boot.scala

```diff
@@ -72,7 +73,8 @@ object Boot extends App with Directives with BootMigrations {
         pathPrefix("api" / "v1") {
           ErrorHandler.handleErrors {
             new Routes(authNamespace, messageBus).route ~
-            new GroupsResource(authNamespace).route
+            new GroupsResource(authNamespace).route ~
+            new SystemInfoResource(authNamespace).route
```

Comment:
Instead of duplicating the routes we could use the Routing class (we do so in resolver)

    new Routing(authNamespace, messageBus).route ~ 
    new HealthResource(db, org.genivi.sota.device_registry.BuildInfo.toMap).route